src/scaffold/template/base/package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "{{ npmPackage }}",
   "version": "0.1.0-rc.1",
+  "majorVersion": "1",
```

Comment:
Huh?

I cannot find any mention of property `majorVersion` in [package.json-schema](http://json.schemastore.org/package). Looks as this file is no longer valid as such a property isn't allowed by schema. (Or did I miss something here?)

Isn't the 1st number in [version](https://docs.npmjs.com/files/package.json#version) the thing we call [major version](https://semver.org/spec/v2.0.0.html#summary)?

Reply:
I removed it a couple of minutes after the PR was merged :) see: https://github.com/swisspush/apikana/pull/125/files.

To answer your question, yes, but we decided for version which starts with 0, the major version number is 1.